qutebrowser/utils/log.py

```diff
@@ -236,6 +242,11 @@ def _init_handlers(level, color, force_color, json_logging, ram_capacity):
     return console_handler, ram_handler
 
 
+def change_loglevel(level):
+    value = LOG_LEVELS[level.upper()]
+    console_handler.setLevel(value)
```

Comment:
I'd do this inside the command, no point in adding a new function in `log.py` as it isn't needed anywhere else.